lib/concurrent/utility/engine.rb

```diff
@@ -20,7 +20,7 @@ def on_rbx?
       end
 
       def on_truffle?
-        ruby_engine == 'jruby+truffle'
+        ruby_engine == 'jruby+truffle' || ruby_engine == 'truffleruby'
```

Comment:
I think we should just update to `ruby_engine == 'truffleruby'`, there seems to be little point to support ancient (and no longer download-able) TruffleRuby versions.